asllib/doc/AbstractSyntax.tex

```diff
@@ -858,7 +858,7 @@ \section{Building Parameterized Productions\label{sec:BuildingParameterizedProdu
 \inferrule[some]{
   b(\vv) \astarrow \astversion{\vv}
 }{
-  \buildoption[b](\overname{\namednode{\vv}{N}}{\vsym}) \astarrow \overname{\Option{\astversion{\vv}}}{\vsymast}
+  \buildoption[b](\overname{\namednode{\vv}{N}}{\vsym}) \astarrow \overname{\Option(\astversion{\vv})}{\vsymast}
```

Comment:
Am I right in thinking that `\some` takes an argument but `\Option` does not? If so, is there a particular reason for this?